### PR TITLE
Arcade machines dispense minibomb as a prize

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Changes the prize for winning against an emagged arcade machine to be a syndicate minibomb.

## How This Contributes To The Skyrat Roleplay Experience

Requested we not hand out the plasma TTV.

## Changelog

:cl: LT3
balance: emagged arcade machines now provide a syndicate minibomb as a prize
/:cl:

### CODE REVIEW

**Comment by Zonespace27** [comment]
Heavily against for how silly strong the ebows are, being able to get what is basically an infinite amount for a few TC is pretty broken

**Comment by SomeRandomOwl** [comment]
Yea i agree with zone here, Ebows are just a one hit knockdown and about a two hit stam crit? I think? (might be three i forget) 

This would be almost too strong of a reward comparatively with how easy it is to beat even a emagged Orion trail, even if the Orion trail has a chance to gib you (or even RR not sure, it can make the the localized short live singularity) this reward is fairly achievable

**Comment by lessthnthree** [comment]
We'll take the discussion to discord on an alternative item, it doesn't **have** to be an ebow.

File: code/game/machinery/computer/arcade/arcade.dm [hunk 1]
```diff
@@ -556,10 +556,11 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 			playsound(loc, 'sound/arcade/win.ogg', 50, TRUE)
 
 			if(obj_flags & EMAGGED)
-				new /obj/effect/spawner/newbomb/plasma(loc, /obj/item/assembly/timer)
+				// new /obj/effect/spawner/newbomb/plasma(loc, /obj/item/assembly/timer) - ORIGINAL
+				new /obj/item/gun/energy/recharge/ebow(loc) // SKYRAT EDIT CHANGE - Weapon instead of bomb
```
Comment (Iamgoofball) [inline code/game/machinery/computer/arcade/arcade.dm]: Swap it for a minibomb instead, it's far more fitting for winning against this arcade.

File: code/game/machinery/computer/arcade/orion.dm [hunk 2]
```diff
@@ -479,9 +479,10 @@ GLOBAL_LIST_INIT(orion_events, generate_orion_events())
 	gameStatus = ORION_STATUS_START
 	say("Congratulations, you made it to Orion!")
 	if(obj_flags & EMAGGED)
-		new /obj/item/orion_ship(loc)
-		message_admins("[ADMIN_LOOKUPFLW(usr)] made it to Orion on an emagged machine and got an explosive toy ship.")
-		usr.log_message("made it to Orion on an emagged machine and got an explosive toy ship.", LOG_GAME)
+		// new /obj/item/orion_ship(loc) - ORIGINAL
+		new /obj/item/gun/energy/recharge/ebow(loc) //SKYRAT EDIT CHANGE - Weapon instead of bomb
```
Comment (Iamgoofball) [inline code/game/machinery/computer/arcade/orion.dm]: Make it give like, idk, some rare modsuit part. Maybe Kinesis?

**Review by tf-4 — CHANGES_REQUESTED** [review]
gonna just make it more official that an ebow is not gonna work here. I don't think other suggestions I've seen of eswords, or other weapons are a good idea for the same reasons.

**Comment by lessthnthree** [comment]
> gonna just make it more official that an ebow is not gonna work here. I don't think other suggestions I've seen of eswords, or other weapons are a good idea for the same reasons.

Then I'll adjust the PR to be simply the host request to reduce the bomb size.

**Pull request closed without merging**